### PR TITLE
Add create-imagemosaic-datastore worker

### PR DESCRIPTION
**Pull request by hblitza**
This updates the `docker-compose.yml` file by adding the `geoserver-create-imagemosaic-datastore` worker.  
Also adapts the respective job jsons.  

@weskamm @chrismayer what do you think?

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -146,6 +146,23 @@ services:
     - GEOSERVER_USER=${GEOSERVER_USER}
     - GEOSERVER_PASSWORD=${GEOSERVER_PASSWORD}
 
+  geoserver-create-imagemosaic-datastore:
+    image: ghcr.io/klips-project/mqm-worker/geoserver-publish-imagemosaic:latest
+    restart: unless-stopped
+    volumes:
+      - ./geoserver_data:/opt/geoserver_data/:Z
+    depends_on:
+      - rabbitmq
```
Comment (weskamm) [inline docker-compose.yml:155]: could add `geoserver`

File: workflows/publish-imagemosaic.json [hunk 2]
```diff
@@ -5,19 +5,28 @@
         "type": "download-file",
         "inputs": [
             "http://nginx/sample.tif",
-            "/opt/geoserver_data/sample.tif"
+            "/opt/geoserver_data/20220101T0000.tif"
```
Comment (weskamm) [inline workflows/publish-imagemosaic.json:8]: please add that file to https://github.com/klips-project/klips-sdi/tree/main/mocked-webspace

**Review by weskamm — COMMENTED** [review]
2 remarks

File: workflows/publish-imagemosaic.json (hunk 2 above)
Comment (hblitza) [reply workflows/publish-imagemosaic.json:8]: no need i think. sample.tif will be downloaded and saved as `20220101T0000.tif`.

Comment (hblitza) [reply workflows/publish-imagemosaic.json:8]: please confirm.

Comment (weskamm) [reply workflows/publish-imagemosaic.json:8]: haha ok, tomatoes on the eyes... :+1:

**Review by weskamm — APPROVED** [review]

**Pull request merged**